dataprofiler/labelers/column_name_model.py

```diff
@@ -217,11 +218,12 @@ def predict(
             include_label=self._parameters["include_label"],
         )
 
-        if show_confidences:
+        if show_confidences is False:
             raise NotImplementedError(
                 """`show_confidences` parameter is disabled
-                for Proof of Concept implementation. Confidence
-                values are enabled by default."""
+                for MVP implementation. Due to the requirement
+                of having the data point in the post processor.
+                Note: Confidence values are returned by default."""
```

Comment:
the post processor requires having the confidence / similarity scores to filter the model results. So currently as built there isn't the option for filtering confidence / similary scores at the model component level